public/index.html

```diff
@@ -45,108 +45,6 @@
-      registration.paymentManager.userHint = "test@bobbucks.dev";
-      return Promise.all([
-        registration.paymentManager.instruments.set(
-            "dc2de27a-ca5e-4fbd-883e-b6ded6c69d4f",
-            {
-              name: "My Bob Pay Account",
-              icons: [{
-                src:"/pay/bobbucks.png",
-                sizes:"32x32",
-                type:"image/png"}
-              ],
-              method: "https://bobbucks.dev/pay"
-            }),
-        registration.paymentManager.instruments.set(
-            "new-card",
-            {
-                name: "Add a new card to BobBucks",
-                method: "basic-card",
-                capabilities: {
-                    supportedNetworks: ["visa", "mastercard", "amex", "discover"],
-                    supportedTypes: ["credit", "debit", "prepaid"]
-                }
-            }),
-      ]);
-    };
-
-    // Shows/hides the webpage controls for installing/removing the service
-    // worker.
-    function showBobBucksStatus(enabled) {
-        showBobBucksError("");
-        var buttonText = enabled ?
-                'Web App Installed &#9989;' : 'Install Web App';
-        document.getElementById("installbutton").innerHTML = buttonText;
-
-        document.getElementById("installed").classList.toggle(
-                'invisible', !enabled);
-        document.getElementById("notinstalled").classList.toggle(
-                'invisible', enabled);
-    }
-
-    // Shows/clears an error related to the service worker installation.
-    function showBobBucksError(errorMessage) {
-        document.getElementById('bobbuckserror').innerHTML = errorMessage;
-    }
-
-    // Registers the payment app service worker by installing the default
-    // instruments.
-    function registerPaymentAppServiceWorker() {
-      navigator.serviceWorker.register(SERVICE_WORKER_URL).then(function(registration) {
-        if(!registration.paymentManager) {
-          // Payment app capability not available, unregister right away.
-          registration.unregister().then((success) => {});
-          showBobBucksError('Payment app capability not present. Enable flags?');
-          return;
-        }
-
-        addInstruments(registration).then(function() {
-            showBobBucksStatus(/*enabled=*/true);
-        });
-      }).catch((error) => {
-        showBobBucksError(error);
-      });
-    }
-
-    // Registers the payment app service worker by installing the default
-    // instruments.
-    function unregisterPaymentAppServiceWorker() {
-      navigator.serviceWorker.getRegistration(SERVICE_WORKER_URL).then(function(registration) {
-        registration.unregister().then((success) => {
-            showBobBucksStatus(!success);
-        });
-    });
-    }
-
-    // When page is loaded, checks for the existence of the service worker.
-    navigator.serviceWorker.getRegistration(SERVICE_WORKER_URL).then(function(registration) {
```

Comment:
Let's keep this part. 

This part checks if the Web Payment Handler is installed, without involving payment instruments. It's a good sample of how to check if a Web Payment Handler is installed.